docker/dev/push_image.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+# Usage: examples
+#    ./push_image.sh
+#    BRANCH_HEAD_TAG='git describe' ./push_image.sh
+#    REPO_OWNER=stanislavchlebec BRANCH_HEAD_TAG=`git describe` ./push_image.sh
+#    LOCAL_IMAGE='prod_vpp_agent:latest' DEFAULT_IMAGE_NAME='vpp-agent' ./push_image.sh
+
+# Warning: use only IMMEDIATELY after docker/dev/build.sh to prevent INCONSISTENCIES such as 
+#          a) after building image you switch to other branch which will result in mismatch of version of image and its tag
+#          b) you do not build the new image but only simply run this script which will result in mismatch version of image and its tag because the image is older than repository 
+
+set -e
+
+# detect branch name
+BRANCH_NAME="$(git symbolic-ref HEAD 2>/dev/null)" || BRANCH_NAME="(unnamed branch)"     # detached HEAD
+BRANCH_NAME=${BRANCH_NAME##refs/heads/}
+BRANCH_HEAD_TAG=${BRANCH_HEAD_TAG:-"`git name-rev --name-only --tags HEAD`"}
+VERSION=$(git describe --always --tags --dirty)
+
+LOCAL_IMAGE=${LOCAL_IMAGE:-'dev_vpp_agent:latest'}
+REPO_OWNER=${REPO_OWNER:-'ligato'}
+DEFAULT_IMAGE_NAME=${DEFAULT_IMAGE_NAME:-'dev-vpp-agent'}
```

Comment:
And as a final thing, just shorten this to `IMAGE_NAME` since we don't have that variable now and I'll merge it.

Reply:
Done:-)
Now it is almost perfect.